sources/osgShader/Compiler.js

```diff
@@ -1165,6 +1477,7 @@ define( [
 
             var fragColor = factory.getNode( 'FragColor' );
```

Comment:
if we name gl variable with the prefix, maybe FragColor should follow the same pattern. Or we can remove gl prefix for other node, but it's better to make it consistent.